mappings/net/minecraft/world/BlockBox.mapping

```diff
@@ -0,0 +1,26 @@
+CLASS net/minecraft/class_156 net/minecraft/world/BlockBox
```

Comment:
`BlockBox` or `Box`? Given that the positions are doubles, I believe that they can take up a different size than a block.